game/entities/sdDrone.js

```diff
@@ -168,7 +183,7 @@ class sdDrone extends sdEntity
 
 	CollisionFiltering( from_entity )
 	{
-		if ( from_entity._is_bg_entity !== this._is_bg_entity || !from_entity._hard_collision )
+		if ( from_entity.IsBGEntity() !== this.IsBGEntity() || !from_entity._hard_collision )
```

Comment:
I'm not sure if we need to bring back `IsBGEntity`, could it be that you have done this on an old version of SD2D? Because that's all I could think of